packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -31,15 +31,21 @@ async function main() {
 
     const packageConf = JSON.parse(await readFile("package.json"));
     const widgetVersion = packageConf?.version;
-    const widgetMpk = ls(`dist/${widgetVersion}/*.mpk`).length;
 
-    if (!widgetMpk) {
-        throw new Error("No widgets founds in dist folder. Please execute `npm run release` before start e2e tests.");
-    }
+    // TODO [https://mendix.atlassian.net/browse/WT-3111]: Use this flag via `"test:e2e": "pluggable-widgets-tools test:e2e:web --skip-mpk-check"` in atlas package.json
```

Comment:
This comment is unrelated to PIW tools.

Reply:
Removed the todo since we have a story now, hopefully we will not forget